gradle-enterprise-api-kotlin.json

```
@@ -1,5 +1,5 @@
 {
-  "description": "A library to use the Gradle Enterprise API in Kotlin scripts or projects",
+  "description": "Deprecated. Use develocity-api-kotlin instead.",
```

It will appear [here](https://github.com/Kotlin/kotlin-jupyter?tab=readme-ov-file#list-of-supported-libraries). I find it more appropriate to leave the description itself, wdyt? 

```suggestion
  "description": "[Deprecated. Use develocity-api-kotlin instead.] A library to use the Gradle Enterprise API in Kotlin scripts or projects",
```

---

Added the description back and made it a little shorter then. Although unless it's an automation that adds it, I'd actually remove `gradle-enterprise-api-kotlin` from the README.

```diff
-  "description": "Deprecated. Use develocity-api-kotlin instead.",
+  "description": "[Deprecated: use develocity-api-kotlin] A library to use the Gradle Enterprise API in Kotlin scripts or projects",
```